aiida_optimize/engines/_convergence.py

```diff
@@ -187,15 +187,16 @@ def _update(self, outputs: ty.Dict[int, ty.Any]) -> None:
             for val in output_values
         ]  # Values are AiiDA types
 
-    def _get_optimal_result(self) -> ty.Tuple[int, Result]:
+    def _get_optimal_result(self) -> ty.Tuple[int, ty.Any, Result]:
```

Comment:
I think the return type here should be `[int, orm.Node, orm.Node]`. The `Result` was already wrong before I think, not sure why `mypy` didn't complain here.